### PR TITLE
feat(openshift-etcd-backup): dockerize backup script and add few tweaks

### PR DESCRIPTION
**Pull request by vmaillot**
* Dockerize backup script with ubi8 image (minimal one does not have `find`).
* Upgrade `cronjob` with `configmap` tweaks
* Update readme

### CODE REVIEW

**Review by hairmare — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi8@sha256:77623387101abefbf83161c7d5a0378379d0424b2244009282acb39d42f1fe13
```
[the remark this answers is not in the document]
Comment (vmaillot) [reply Dockerfile]: Thanks for the hint on how to install `find`!

Yes after some tests, the script works well and let us backup etcd. Why would we need those scripts then? To have a `standard` and `official` way to backup etcd? So we should add `cluster-backup.sh` scripts and remove our `backup.sh` script?

[the remark this answers is not in the document]
Comment (hairmare) [reply Dockerfile]: afaict ours is a wrapper

https://github.com/adfinis-sygroup/openshift-etcd-backup/blob/6b2cdde031a960ffdb5cd2505930240795b94da0/backup.sh#L63

[the remark this answers is not in the document]
Comment (vmaillot) [reply Dockerfile]: but why doing so as it's already on the node itself? to get latest version?

[the remark this answers is not in the document]
Comment (hairmare) [reply Dockerfile]: oh, i missed that it's being grabbed from the hostPath `/` volume :stuck_out_tongue:

**Review by hairmare — APPROVED** [review]

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (vmaillot) [reply Dockerfile]: yay exactly, thanks!

**Pull request merged**